codex.nim

```diff
@@ -67,6 +67,9 @@ when isMainModule:
     # permissions are insecure.
     quit QuitFailure
 
+  if not(checkAndCreateDataDir((config.metaDir).string)):
+    quit QuitFailure
```

Comment:
An error message before quitting would be helpful.

Reply:
Yes. I'll also add it to the places where I copy-pasted it from.